docs/src/manual.md

```diff
@@ -12,9 +12,17 @@ The [RDatasets package](https://github.com/johnmyleswhite/RDatasets.jl) is usefu
 
 ## Fitting GLM models
 
-To fit a Generalized Linear Model (GLM), use the function, `glm(formula, data, family, link)`, where,
-- `formula`: uses column symbols from the DataFrame data, for example, if `names(data)=[:Y,:X1,:X2]`, then a valid formula is `@formula(Y ~ X1 + X2)`
-- `data`: a DataFrame which may contain NA values, any rows with NA values are ignored
+Two methods can be used to fit a Generalized Linear Model (GLM):
+`glm(formula, data, family, link)` and `glm(X, y, family, link)`.
```

Comment:
I hadn't realized that the order was `X`, `y`. This is completely counter-intuitive to me. :-/

Reply:
Yeah I agree it's counter intuitive.  It would also be hugely breaking to change so...

Reply:
I think this was done to match the argument order in Python's StatsModels and a lot of "rawer" R packages.

Reply:
That explains it. :-)